apps/paramaps/itkimage2paramap.cxx

```diff
@@ -5,6 +5,13 @@
 #undef HAVE_SSTREAM // Avoid redefinition warning
 #include "dcmqi/ParaMapConverter.h"
 
+#ifdef _WIN32
+#include "dirent_win.h"
+#else
+#include <dirent.h>
+#endif
+
+
```

Comment:
@fedorov maybe we should use the dcmtk provided function for searching directories for files?